lms/static/scripts/frontend_apps/components/UIMessage.tsx

```diff
@@ -0,0 +1,46 @@
+import { CancelIcon, CheckIcon } from '@hypothesis/frontend-shared';
+import classnames from 'classnames';
+import type { ComponentChildren, JSX } from 'preact';
+
+export type UIMessageProps = {
+  status?: 'success' | 'error' | 'info';
+  children: ComponentChildren;
+} & JSX.HTMLAttributes<HTMLDivElement>;
+
+/**
+ * Style a UI message associated with content-picking user inputs
```

Comment:
This was the best description I could come up with but it felt a little weak so I'm open to feedback.